augeias/collections/__init__.py

```diff
@@ -4,11 +4,15 @@
 from augeias.stores.PairTreeFileSystemStore import PairTreeFileSystemStore
 
 
-def includeme(config):   # pragma: no cover
-    beeldbank = Collection(name='cheeses', object_store=PairTreeFileSystemStore(
-        os.path.expanduser('~/data/cheeses/data')))
+def includeme(config):  # pragma: no cover
```

Comment:
nu we hier toch zitten, zou je beeldbank kunnen hernoemen naar cheeses, en besluiten naar trees